src/pcm.rs

```diff
@@ -554,6 +592,10 @@ impl<'a> HwParams<'a> {
             .and_then(|_| Format::from_c_int(v, "snd_pcm_hw_params_get_format"))
     }
 
+    pub fn test_format(&self, v: Format) -> bool {
```

Comment:
Please add a short comment like "returns true if the format is available" (just because it differs from what alsa-lib does)